HW03/matsvei_basiayeu/matsvei_basiayeu_hw_03_t_01.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(log)
+    log.split("\n").select { |message| message.include?('error') }.first.to_s
```

Comment:
```suggestion
    log.split("\n").find { |message| message.include?('error') }.to_s
```
Could you read about the difference between `find` and `select` and write you answer in my telegram direct please?

Reply:
Ok, fixed, thank you!